blazor/calendar/style-appearance.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: Style and appearance in Blazor Calendar Component | Syncfusion
+description: Checkout and learn here all about Style and appearance in Syncfusion Blazor Calendar component and more.
+platform: Blazor
+control: Calendar
+documentation: ug
+---
+
+# Style and appearance in Blazor Calendar Component
+
+The following content provides the exact CSS structure that can be used to modify the control's appearance based on the user preference.
+
+## Customizing the Calendar
+
+Use the following CSS to customize the Calendar.
```

Comment:
Use the description to add details to what css properties can be customized here in calendar